deltachat-jsonrpc/src/api/types/qr.rs

```diff
@@ -304,3 +305,53 @@ impl From<Qr> for QrObject {
         }
     }
 }
+
+#[derive(Deserialize, TypeDef, schemars::JsonSchema)]
+pub enum SecurejoinSource {
+    /// Because of some problem, it is unknown where the QR code came from.
+    Unknown = 0,
+    /// The user opened a link somewhere outside Delta Chat
+    ExternalLink = 1,
+    /// The user clicked on a link in a message inside Delta Chat
+    InternalLink = 2,
+    /// The user clicked "Paste from Clipboard" in the QR scan activity
+    Clipboard = 3,
+    /// The user clicked "Load QR code as image" in the QR scan activity
+    ImageLoaded = 4,
+    /// The user scanned a QR code
+    Scan = 5,
+}
+
+#[derive(Deserialize, TypeDef, schemars::JsonSchema)]
+pub enum SecurejoinUIPath {
```

Comment:
`SecurejoinUiPath`. Abbreviations are usually camelcased being parts of another name. See `QrIcon` below